testground/benchmark/benchmark/stateless.py

```diff
@@ -146,12 +146,14 @@ def run(
         proc.wait()
 
         # collect outputs
+        output = Path("/data.tar.bz2")
+        with tarfile.open(output, "x:bz2") as tar:
+            tar.add(home, arcname="data")
         outdir = Path(outdir)
         if outdir.exists():
```

Comment:
should we check like is_dir?

Reply:
I think it's fine, it'll raise error if the mount is not a directory.

Reply:
I see, but just get `IsADirectoryError: [Errno 21] Is a directory: '/outputs'` in `filename.unlink(missing_ok=True)`

Reply:
added an specific assert